utils/utils.go

```diff
@@ -41,6 +41,9 @@ const (
 	DOCKER ContainerRuntime = "docker"
 	PODMAN ContainerRuntime = "podman"
 
+	DefaultComponentsDirName = "components"
```

Comment:
Is the `default` prefix needed here? This is not a configurable option anyway, right?

Reply:
I think you are right. We can change it to `DaprComponentsDirName`?

Reply:
Ok, so `dapr init` always installs at `components`/`resources`, but overrides can happen via `dapr run`. So it makes sense to keep it as default then.